src/adapters/EnzymeAdapter.js

```diff
@@ -0,0 +1,24 @@
+function unimplementedError(methodName, classname) {
+  return new Error(
+    `${methodName} is a required method of ${classname}, but was not implemented.`,
+  );
+}
+
+class EnzymeAdapter {
```

Comment:
Whats the strategy for exposing this class for adapter authors? Will it just be accessible from a `lib` folder like

```js
import EnzymeAdapter from 'enzyme/lib/EnzymeAdapter'
```